demoqa/src/test/java/demoqa/com/ControlExtensionBase.java

```diff
@@ -0,0 +1,9 @@
+package demoqa.com;
+
+import org.openqa.selenium.WebElement;
+
+public abstract class ControlExtensionBase {
```

Comment:
You should add a constructor here that initializes wrappedElement.

Reply:
Resolved.